src/lobby/client.ts

```diff
@@ -1,4 +1,25 @@
 import type { LobbyAPI } from '../types';
+import { Cartesify } from '@calindra/cartesify';
+/**
+ * Creates a Cartesify fetch instance with the specified DAPP address and endpoints.
+ *
+ *
+ * @param {string} DAPP_ADDRESS - The address of the DAPP to use with the Cartesify fetch instance.
+ * @param {Object} endpoints - An object containing the URLs for the GraphQL and inspect endpoints.
+ * @param {URL} endpoints.graphQL - The URL for the GraphQL endpoint.
+ * @param {URL} endpoints.inspect - The URL for the inspect endpoint.
+ * @returns {Object} - A Cartesify fetch instance configured with the provided DAPP address and endpoints.
+ */
+
+const DAPP_ADDRESS = '0xab7528bb862fb57e8a2bcd567a2e929a0be56a5e'; //Review
+
+const cartesifyFetch = Cartesify.createFetch({
+  dappAddress: DAPP_ADDRESS,
+  endpoints: {
+    graphQL: new URL('http://localhost:8080/graphql'),
+    inspect: new URL('http://localhost:8080/inspect'),
```

Comment:
suggestion: Use defined server in these endpoints

Once we move the fetch to the class, we probably can use the server field as part of this endpoints definitions, for example: 

```suggestion
    inspect: new URL(`${this.server}/inspect`),
```